### PR TITLE
Portfolio: accessibility

### PR DESCRIPTION
**Pull request by AJ193**
On this milestone we have tried to meet the specified accessibility requirements.Which are:

- Page titles - no issues found
- image alt - (link = https://github.com/AJ193/Portfolio-A/pull/8/commits/a4bd713e4a0b4d72cf0f43a7aa1978b7745eec33)
- Headings - no issues found.
- Color - No issue because we have implemented figma's design.
- Resize - no issues found.
- labels - (link = https://github.com/AJ193/Portfolio-A/pull/8/commits/a4bd713e4a0b4d72cf0f43a7aa1978b7745eec33)
- moving - no issues found.
- Multimedia - no issues found.
- Basic structure - no issues found.

### CODE REVIEW

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi team, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 


## Required Changes ♻️
- According to the project requirements, `If issues were found, they should link to the corresponding line of code in the Pull Request changes`. Please fix this for all changes made 🙏 
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -62,7 +62,7 @@ <h3 class="underlined">My Recent Works <span class="underlined-hr"></span> </h3>
         <div class="project-list">
           <div class="project-item">
             <div class="project-item-img">
-          <img src="./images/IMAGE5.PNG.png" alt="" class="img">
+          <img src="./images/IMAGE5.PNG.png" alt="yoga" class="img">
```
Comment (kifle23) [inline index.html:65]: An alt message is added

File: index.html [hunk 2]
```diff
@@ -350,22 +350,27 @@ <h3 class="contact-heading">
...
+      <label for="fullname" hidden></label>
+      <input type="text" name="fullname" class="contact-input-1" placeholder="Full Name" maxlength="30" required>
+      <label for="firstname" hidden></label>
+      <input type="text" name="firstname" class="contact-input-2" placeholder="First name" maxlength="30" required>
+      <label for="lastname" hidden></label>
+      <input type="text" name="lastname" class="contact-input-3" placeholder="last name" maxlength="30" required>
+      <label for="email" hidden></label>
+      <input type="email" name="email" class="contact-input-4" placeholder="Email address" required>
+      <label for="text" hidden></label>
+      <textarea name="textarea" name="text" class="contact-textarea" id="" cols="30" rows="10" required maxlength="500" placeholder="Write me Something..."></textarea>
```
Comment (kifle23) [inline index.html:362]: Labels are added to the contact form elements

File: index.html [hunk 3]
```diff
@@ -350,22 +350,27 @@ <h3 class="contact-heading">
...
-        <li class="footer-social-item"><a href="https://github.com/AJ193"></a><img src="images/Disabled-G.png"></li>
-        <li class="footer-social-item"><img src="images/Disabled-A.png"></li>
-        <li class="footer-social-item"><a href="https://www.linkedin.com/in/ghulam-shabbir-225264247/"></a><img src="images/Disabled-L.png"></li>
-        <li class="footer-social-item"><a href="https://twitter.com/GhulamShabbir59"></a><img src="images/Disabled-T.png"></li>
-        <li class="footer-social-item"><img src="images/Disabled-M.png"></li>
+        <li class="footer-social-item"><a href="https://github.com/AJ193"></a><img src="images/Disabled-G.png" alt="github"></li>
+        <li class="footer-social-item"><img src="images/Disabled-A.png" alt="Angelis"></li>
+        <li class="footer-social-item"><a href="https://www.linkedin.com/in/ghulam-shabbir-225264247/"></a><img src="images/Disabled-L.png" alt="Linkedin"></li>
+        <li class="footer-social-item"><a href="https://twitter.com/GhulamShabbir59"></a><img src="images/Disabled-T.png" alt="twitter"></li>
+        <li class="footer-social-item"><img src="images/Disabled-M.png" alt="Medium"></li>
```
Comment (kifle23) [inline index.html:373]: We have added an alt message for images

File: index.html [hunk 4]
```diff
@@ -46,11 +46,11 @@ <h2 class="body-text-header">I'm a Software Developer</h2>
...
-        <li class="portfolio-social-item"><a href="https://www.linkedin.com/in/ghulam-shabbir-225264247/"><img src="images/Linkedin.png"></a></li>
-        <li class="portfolio-social-item"><a href="#"><img src="images/angelis.png"></a></li>
-        <li class="portfolio-social-item"><a href="https://twitter.com/GhulamShabbir59"><img src="images/Twitter.png"></a></li>
-        <li class="portfolio-social-item"><a href="#"><img src="images/Microsoft.png"></a></li>
+        <li class="portfolio-social-item"><a href="https://github.com/AJ193"><img src="images/Github.png" alt="github"></a></li>
+        <li class="portfolio-social-item"><a href="https://www.linkedin.com/in/ghulam-shabbir-225264247/"><img src="images/Linkedin.png" alt="linkedin"></a></li>
+        <li class="portfolio-social-item"><a href="#"><img src="images/angelis.png" alt="angelis"></a></li>
+        <li class="portfolio-social-item"><a href="https://twitter.com/GhulamShabbir59"><img src="images/Twitter.png" alt="twitter"></a></li>
+        <li class="portfolio-social-item"><a href="#"><img src="images/Microsoft.png" alt="medium"></a></li>
```
Comment (kifle23) [inline index.html:53]: We have added an alt message for images

File: index.html (hunk 1 above)
Comment (mckent05) [reply index.html:65]: - Yes, you have done well. But please add a link for each of the changes you have made to the PR report you have created,

**Comment by AJ193** [comment]
@mckent05 Can you please recheck we have added the links to the pull request.

File: index.html [hunk 5]
```diff
@@ -350,22 +350,27 @@ <h3 class="contact-heading">
         I'm always interested<br>in hearing about new<br>projects,so if you'd<br>like to chat please<br>get in touch.
       </h3>
     <form action="https://formspree.io/f/xgebvjyw" method="POST" class="contact-form">
-      <input type="text" class="contact-input-1" placeholder="Full Name" maxlength="30" required>
-      <input type="text" class="contact-input-2" placeholder="First name" maxlength="30" required>
-      <input type="text" class="contact-input-3" placeholder="last name" maxlength="30" required>
-      <input type="email" class="contact-input-4" placeholder="Email address" required>
-      <textarea name="textarea" class="contact-textarea" id="" cols="30" rows="10" required maxlength="500" placeholder="Enter text here"></textarea>
+      <label for="fullname" hidden></label>
+      <input type="text" name="fullname" class="contact-input-1" placeholder="Full Name" maxlength="30" required>
```
Comment (LionRouge1) [inline index.html:354]: [OPTIONAL]
- [ ] Kindly add `id="fullname"` attribute to this input to make it matches the `label` element with a `for="fullname"`. That will make your label become clickable, increasing the target area and making it easier to select small radio buttons or checkboxes.

File: index.html [hunk 6]
```diff
@@ -350,22 +350,27 @@ <h3 class="contact-heading">
...
     <form action="https://formspree.io/f/xgebvjyw" method="POST" class="contact-form">
-      <input type="text" class="contact-input-1" placeholder="Full Name" maxlength="30" required>
-      <input type="text" class="contact-input-2" placeholder="First name" maxlength="30" required>
-      <input type="text" class="contact-input-3" placeholder="last name" maxlength="30" required>
-      <input type="email" class="contact-input-4" placeholder="Email address" required>
-      <textarea name="textarea" class="contact-textarea" id="" cols="30" rows="10" required maxlength="500" placeholder="Enter text here"></textarea>
+      <label for="fullname" hidden></label>
+      <input type="text" name="fullname" class="contact-input-1" placeholder="Full Name" maxlength="30" required>
+      <label for="firstname" hidden></label>
+      <input type="text" name="firstname" class="contact-input-2" placeholder="First name" maxlength="30" required>
```
Comment (LionRouge1) [inline index.html:356]: [OPTIONAL]
- [ ] Same here too :point_up_2: But `id="firstname"`

File: index.html [hunk 7]
```diff
@@ -350,22 +350,27 @@ <h3 class="contact-heading">
...
+      <label for="lastname" hidden></label>
+      <input type="text" name="lastname" class="contact-input-3" placeholder="last name" maxlength="30" required>
```
Comment (LionRouge1) [inline index.html:358]: [OPTIONAL]
- [ ] Same here too :point_up_2: But `id="lastname"`

File: index.html [hunk 8]
```diff
@@ -350,22 +350,27 @@ <h3 class="contact-heading">
...
+      <label for="email" hidden></label>
+      <input type="email" name="email" class="contact-input-4" placeholder="Email address" required>
```
Comment (LionRouge1) [inline index.html:360]: [OPTIONAL]
- [ ] Same here too :point_up_2: But `id="email"`

File: index.html [hunk 9]
```diff
@@ -350,22 +350,27 @@ <h3 class="contact-heading">
...
+      <label for="text" hidden></label>
+      <textarea name="textarea" name="text" class="contact-textarea" id="" cols="30" rows="10" required maxlength="500" placeholder="Write me Something..."></textarea>
```
Comment (LionRouge1) [inline index.html:362]: [OPTIONAL]
- [ ] Same here too :point_up_2: But `id="text"`

**Review by LionRouge1 — APPROVED** [review]
Hi @AJ193, 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well evaluating your website's accessibility to find accessibility issues and fix them :+1: 

done

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**